app/src/org/commcare/views/widgets/RecordingFragment.java

```diff
@@ -196,8 +205,15 @@ private void setupRecorder() {
         }
 
         boolean isHeAacSupported = isHeAacEncoderSupported();
-
-        recorder.setAudioSource(MediaRecorder.AudioSource.MIC);
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
+            recorder.setAudioSource(MediaRecorder.AudioSource.VOICE_COMMUNICATION);
+       } else {
+            recorder.setAudioSource(MediaRecorder.AudioSource.MIC);
+       }

```

Comment:
We should test that the recording with source set to `VOICE_COMMUNICATION` are playable in browser (instead of download) on HQ.

Reply:
Any updates here ?

Reply:
Yes, I tested it and it plays normally.